custom_functions/fast_etl.py

```diff
@@ -232,6 +241,75 @@ def compare_source_dest_rows(source_cur,
                       f'Origem: {source_row_count} linhas. ' \
                       f'Destino: {destination_row_count} linhas')
 
+
+def get_hook_and_engine_by_provider(
+        provider: str,
+        conn_id: str) -> [DbApiHook, Engine]:
+    if provider == 'MSSQL':
+        hook = MsSqlHook(conn_id)
+        engine = get_mssql_odbc_engine(conn_id)
+    elif provider == 'PG':
+        hook = PostgresHook(conn_id)
+        engine = hook.get_sqlalchemy_engine()
+    elif provider == 'MYSQL':
+        hook = MySqlHook(conn_id)
+        engine = hook.get_sqlalchemy_engine()
+
+    return hook, engine
+
+
+def _convert_column(old_col: Column, db_provider: str) -> Column:
+    type_mapping = {
+        'NUMERIC': sa_types.Numeric(38, 13),
+        'BIT': sa_types.Boolean(),
+    }
+    if db_provider.upper() == 'MSSQL':
+        type_mapping['DATETIME'] = sa_dialects.mssql.DATETIME2()
+
+    return Column(old_col['name'],
+                  type_mapping.get(
+                      str(old_col['type']._type_affinity()),
+                      old_col['type']._type_affinity()))
+
+
+def create_table_if_not_exist(source_table: str,
+                              source_conn_id: str,
+                              source_provider: str,
+                              destination_table: str,
+                              destination_conn_id: str,
+                              destination_provider: str
+                              ) -> None:
+    _, source_eng = get_hook_and_engine_by_provider(
+        source_provider, source_conn_id)
+    destination_hook, destination_eng = get_hook_and_engine_by_provider(
+        destination_provider, destination_conn_id)
+    try:
+        destination_hook.get_pandas_df(
+            f'select * from {destination_table} where 1=2')
+    except DatabaseError:
```

Comment:
```suggestion
    except DatabaseError as db_error:
```

Sugiro verificar a mensagem de erro também. Pois a exceção `DatabaseError` pode também vir de outro erro que não seja a falta da tabela.